fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -314,8 +315,20 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 		double readValueTimeout;
 		double readValuePrefixTimeout;
 		double readRangeTimeout;
-
-		explicit Reader(DB& db) : db(db) {
+		Reference<Histogram> readValueDist;
```

Comment:
Please add a comment about thread safety. In particular, it is now only safe to construct a `Reader` on the main thread.

Reply:
Done